person.rb

```diff
@@ -3,13 +3,14 @@
 require './trimerdecorator'
 class Person < Nameble
```

Comment:
Kindly make sure that there's a method for adding rentals. The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association.

For example:
```
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```